pkg/build/client/v1/logs.go

```diff
@@ -0,0 +1,31 @@
+package v1
```

Comment:
Shims? From my comparison it looks like only the parameter differs, so in-place conversions to internals and use that, at least for now.

Reply:
This doesn't satisfy a complexity requirement.  A duplicate here is ok.